common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -143,15 +143,18 @@ public BrokerRequest brokerRequestFromAcquireTokenParameters(@NonNull final Inte
     public BrokerRequest brokerRequestFromSilentOperationParameters(@NonNull final SilentTokenCommandParameters parameters) {
 
         Logger.info(TAG, "Constructing result bundle from AcquireTokenSilentOperationParameters.");
+        final String extraOptions = parameters.getExtraOptions() != null ?
+                QueryParamsAdapter._toJson(parameters.getExtraOptions()) : null;
 
-        final BrokerRequest brokerRequest = new BrokerRequest.Builder()
+        final BrokerRequest brokerRequest = BrokerRequest.builder()
                 .authority(parameters.getAuthority().getAuthorityURL().toString())
                 .scope(TextUtils.join(" ", parameters.getScopes()))
                 .redirect(getRedirectUri(parameters))
+                .extraOptions(extraOptions)
                 .clientId(parameters.getClientId())
                 .homeAccountId(parameters.getAccount().getHomeAccountId())
                 .localAccountId(parameters.getAccount().getLocalAccountId())
-                .username(parameters.getAccount().getUsername())
+                .userName(parameters.getAccount().getUsername())
```

Comment:
Major or minor version change?

Reply:
src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java 

in internal....

Reply:
Resolved offline